flink-runtime/src/main/java/org/apache/flink/runtime/rest/messages/checkpoints/CheckpointConfigInfo.java

```diff
@@ -79,6 +81,9 @@
     @JsonProperty(FIELD_NAME_STATE_BACKEND)
     private final String stateBackend;
 
+    @JsonProperty(FIELD_NAME_CHECKPOINT_STORAGE)
+    private final String checkpointStorage;
```

Comment:
I couldn't find rendering of this field in the UI. 
Or is it implemented in the subsequent PR?

Reply:
Forgot to backport, nice catch.

Reply:
I didn't add it to the UI because the state_backend field is also not rendered in the UI. What do you think?

Reply:
Hmm..you're right. I think it would be nice to show it but can be implemented in a separate PR/ticket.

Reply:
Agreed. I will open a follow-up ticket.